src/Symfony/Component/Notifier/Bridge/Primotexto/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/primotexto-notifier",
+    "type": "symfony-notifier-bridge",
+    "description": "Symfony Primotexto Notifier Bridge",
+    "keywords": ["sms", "primotexto", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Samaël Tomas",
+            "email": "samael.tomas@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "symfony/http-client": "^6.4|^7.0",
+        "symfony/notifier": "^6.4|^7.0"
```

Comment:
```suggestion
        "symfony/notifier": "^7.2"
```

Reply:
Please use 7.2, as we are in feature freeze and this bridge will make it into 7.2